.travis.yml

```diff
@@ -17,15 +17,22 @@ notifications:
   email: false
 
 before_script:
-  - go install github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
I'm not sure why, but this started consistently failing, and switching to this alternative install method fixed it 🤷‍♀ . I plan to switch it back and try it out before merging, but if the problem persists I'll leave it this way